content/en/user-manual/graphics/lighting/lights.md

```diff
@@ -58,11 +56,53 @@ A spot light lights an object like this:
 
 ![Spot light][6]
 
-Spot lights are the most costly type of light so use them sparingly.
+## Light Shapes
+
+There are four light source shapes:
+
+* Punctual
+* Rectangle
+* Disk
+* Sphere
+
+### Punctual
+
+The punctual light source shape is an infinitesimally small point. This is the default light source shape and is a less physically correct, but relatively low cost approximation of a light source. The other light source shapes are more costly to render but will give more correct ligthing and specular reflections. Examples given in [table](#use-cases) below.
```

Comment:
This page uses the phrase `Examples given in [table](#use-cases) below.` several times. I'm not sure this is really necessary.

Reply:
Good point - I will rewrite this to avoid repetition

Reply:
done